release-notes.md

```diff
@@ -1,19 +1,92 @@
+    + [HTTP based API](#http-based-api)
+    + [Memory mapped virtio transport](#memory-mapped-virtio-transport)
+    + [Paravirtualized IOMMU](#paravirtualized-iommu)
+    + [Ubuntu 19.10](#ubuntu-1910)
+    + [Guest large memory](#guest-large-memory)
 - [v0.2.0](#v020)
-	- [Network device offloading](#network-device-offloading)
-	- [Minimal hardware-reduced ACPI](#minimal-hardware-reduced-acpi)
-	- [Debug I/O port](#debug-io-port)
-	- [Improved direct device assignment](#improved-direct-device-assignment)
-	- [Improved shared filesystem](#improved-shared-filesystem)
-	- [Ubuntu bionic based CI](#ubuntu-bionic-based-ci)
+    + [Network device offloading](#network-device-offloading)
+    + [Minimal hardware-reduced ACPI](#minimal-hardware-reduced-acpi)
+    + [Debug I/O port](#debug-i-o-port)
+    + [Improved direct device assignment](#improved-direct-device-assignment)
+    + [Improved shared filesystem](#improved-shared-filesystem)
+    + [Ubuntu bionic based CI](#ubuntu-bionic-based-ci)
 - [v0.1.0](#v010)
-	- [Shared filesystem](#shared-filesystem)
-	- [Initial direct device assignment support](#initial-direct-device-assignment-support)
-	- [Userspace IOAPIC](#userspace-ioapic)
-	- [Virtual persistent memory](#virtual-persistent-memory)
-	- [Linux kernel bzImage](#linux-kernel-bzimage)
-	- [Console over virtio](#console-over-virtio)
-	- [Unit testing](#unit-testing)
-	- [Integration tests parallelization](#integration-tests-parallelization)
+    + [Shared filesystem](#shared-filesystem)
+    + [Initial direct device assignment support](#initial-direct-device-assignment-support)
+    + [Userspace IOAPIC](#userspace-ioapic)
+    + [Virtual persistent memory](#virtual-persistent-memory)
+    + [Linux kernel bzImage](#linux-kernel-bzimage)
+    + [Console over virtio](#console-over-virtio)
+    + [Unit testing](#unit-testing)
+    + [Integration tests parallelization](#integration-tests-parallelization)
+
+# v0.3.0
+
+This release has been tracked through the [0.3.0 project](https://github.com/intel/cloud-hypervisor/projects/3).
+
+Highlights for `cloud-hypervisor` version 0.3.0 include:
+
+### Block device offloading
+
+We continue to work on offloading paravirtualized I/O to external processes,
+and we added support for
+[vhost-user-blk](https://access.redhat.com/solutions/3394851) backends.
+This enables `cloud-hypervisor` users to plug a `vhost-user` based block device
+like [SPDK](https://spdk.io)) into the VMM as their paravirtualized storage
+backend.
+
+### Network device backend
+
+The previous release provided support for
+[vhost-user-net](https://access.redhat.com/solutions/3394851) backends. Now we
+also provide a TAP based vhost-user-net backend, implemented in Rust. Together
+with the vhost-user-net device implementation, this will eventually become the
+Cloud Hypervisor default paravirtualized networking architecture.
+
+### Virtual sockets
+
+In order to more efficiently and securely communicate with the VMM host, we
```

Comment:
s/with the VMM host/between host and guest/

Reply:
Fixed.